recipe/meta.yaml

```
@@ -28,19 +32,28 @@ source:
     folder: vendor/fmt
 
 build:
-  number: 0
+  number: {{ build }}
   skip: true  # [win]
   skip: true  # [py<35]
   detect_binary_files_with_prefix: true
 
+  # add build string so packages can depend on
+  # dagmc or nodagmc variants explicitly:
+  # `openmc * nodagmc_*` for openmc without dagmc
+  # `openmc * dagmc_*` for openmc with dagmc
+
+  string: {{ dagmc }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ build }}
+
 requirements:
   build:
     - cmake
     - make
+    - dagmc # [dagmc == 'dagmc']
```

the linter wants an extra space here, but I have no idea if that's what's causing problems or not???
```suggestion
    - dagmc  # [dagmc == 'dagmc']
```

---

thanks